### ISSUE TITLE
Improvements of the GUI

### ISSUE THREAD

**Issue by AlexHego** [MouseLand/cellpose#623]
Dear all,
First of all I wanted to thank you for your work that has revolutionized image analysis in biology. 

I would like to suggest new features to improve the Cellpose GUI:

1) a correction tool that removes some of the pixels that are not necessary in the mask (like ALT + Click in QuPath)
2) A possibility to load a folder for training, and a second folder with validation data
3) A batch prediction though the GUI (like the serial cellpose napari)  (users sometime are scared by jupyter notebook)
4) Possibility to see during the training the training and validation curves
5) Possibility to check the QCmetrics like in ZeroCostDl4Mic
    """Calculate detection/instance segmentation metrics between ground truth and predicted label images.
    Currently, the following metrics are implemented:
    'fp', 'tp', 'fn', 'precision', 'recall', 'accuracy', 'f1', 'criterion', 'thresh', 'n_true', 'n_pred', 'mean_true_score', 'mean_matched_score', 'panoptic_quality'
    Corresponding objects of y_true and y_pred are counted as true positives (tp), false positives (fp), and false negatives (fn)
    whether their intersection over union (IoU) >= thresh (for criterion='iou', which can be changed)
    * mean_matched_score is the mean IoUs of matched true positives
    * mean_true_score is the mean IoUs of matched true positives but normalized by the total number of GT objects
    * panoptic_quality defined as in Eq. 1 of Kirillov et al. "Panoptic Segmentation", CVPR 2019
    Parameters

6) An annotation tool in 3D with the possibility to load  a Z-stack and copy paste annotations
best,

**Comment by carsen-stringer** [comment]
Thanks Alex! The easiest of these is definitely number 2, and 3 is not too bad either. I like the idea of 4, but at least the user can inspect the command line for this info (so a workaround for now)

For 5 are you referring to the statistics on the validation set? we have all of these implemented in cellpose so we could return them.

For 6, we do in fact allow annotation in 3D, by turning off single stroke. If you press enter a 3D mask is drawn which can interpolate across planes which were not annotated. Was there something missing with this or was the documentation not very clear?

**Comment by AlexHego** [comment]
Dear Carsen, 
regarding point 5, using the human-in-the-loop method can make it difficult for researchers to accurately measure detection accuracy. Utilizing QC metrics can present it in an objective manner.
